tests/multi_ptr/multi_ptr_subscript_op_fp16.cpp

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2023 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "multi_ptr_subscript_op_common.h"
+
+namespace multi_ptr_subscript_op_fp16 {
+
+TEST_CASE("multi_ptr operator[]. fp16 type ", "[multi_ptr]") {
+  using namespace multi_ptr_subscript_op;
```

Comment:
I do not think that it really helps the understanding.

Reply:
Added operator parameter and removed `using namespace`.